site/pages/code-war.rst

```diff
@@ -0,0 +1,40 @@
+.. title: Code War
+.. slug: code-war
+.. date: 2018-06-07 23:33:47 UTC+07:00
+.. tags: 
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+
+
+Rules of the Game
+=================
+
+Eight teams, onstage knockout rounds of short programming bouts, loud crowd..
+mildly impressive prizes. Any language allowed, no holds barred. Think of it
+like cage fighting for coders.
+
+We guarantee this won’t be mind twisting problems for algorithm heads. It will
+help to have a team with a mix of skills, a clever and/or intimidating team
+name and vocal cheer squad.
+
+For example: “Here is the complete text of *Pride and Prejudice* from this URL.
+Be the first to print out the 7th and 25th sentences that contain both
+’Elizabeth’ and ‘Darcy’“.
+
+1. One laptop per team and no internet use at any time. You will be live coding.
```

Comment:
I thought the word "Internet" is always capitalized in English as it's a proper noun?

https://en.oxforddictionaries.com/definition/internet